src/lightning/pytorch/core/module.py

```diff
@@ -1299,22 +1302,23 @@ def to_onnx(self, file_path: Union[str, Path], input_sample: Optional[Any] = Non
             **kwargs: Will be passed to torch.onnx.export function.
 
         Example:
-            >>> class SimpleModel(LightningModule):
```

Comment:
Removed as doctest because it's not easy to skip based on a condition (onnx available)

Reply:
You could convert it to a regular code example, and/or move it to the .rst doc

Reply:
Isn't the current change already a conversion to a regular code example?